include/Gaffer/TweakPlug.inl

```diff
@@ -93,7 +93,12 @@ bool TweakPlug::applyTweak(
 	}
 
 	const IECore::Data *currentValue = getDataFunctor( name );
-	if( currentValue && currentValue->typeId() != newData->typeId() )
+	if(
+		currentValue &&
+		currentValue->typeId() != newData->typeId() &&
+		!( currentValue->typeId() == IECore::StringData::staticTypeId() && newData->typeId() == IECore::InternedStringData::staticTypeId() ) &&
```

Comment:
I don't think we can ever be in a state where `newData` is InternedStringData, because `PlugAlgo::getValueAsData()` will never return it.

Reply:
Fixed in 0c5591ca2a40b281ed6b7b2db82bd52d2d7c40d9